src/main/resources/db/migration/V2_1_SENTENCE_ADJUSTMENTS.sql

```diff
@@ -0,0 +1,38 @@
+create table SENTENCE_ADJUSTMENTS
```

Comment:
this not copied from prison-api so added v2 so the numbers never clash as we copy more files